src/lib/core/overlay/scroll/close-scroll-strategy.ts

```diff
@@ -0,0 +1,38 @@
+import {ScrollStrategy} from './scroll-strategy';
+import {OverlayRef} from '../overlay-ref';
+import {Subscription} from 'rxjs/Subscription';
+import {ScrollDispatcher} from './scroll-dispatcher';
+
+
+/**
+ * Strategy that will close the overlay as soon as the user starts scrolling.
+ */
+export class CloseScrollStrategy implements ScrollStrategy {
+  private _scrollSubscription: Subscription|null = null;
+  private _overlayRef: OverlayRef;
+
+  constructor(private _scrollDispatcher) { }
+
+  attach(overlayRef: OverlayRef) {
+    this._overlayRef = overlayRef;
+  }
+
+  enable() {
+    if (!this._scrollSubscription) {
+      this._scrollSubscription = this._scrollDispatcher.scrolled(null, () => {
+        if (this._overlayRef.hasAttached()) {
+          this._overlayRef.detach();
```

Comment:
Added the observables as discussed. They're not being used anywhere right now since nothing uses the `CloseScrollStrategy`. Also I initially wanted to have the outputs on the `ConnectedOverlayDirective` proxy the ones on the overlay ref, but it's not possible since the ref gets created lazily before attaching.